pinotdb/db.py

```diff
@@ -395,7 +396,15 @@ def finalize_query_payload(
     def normalize_query_response(self, input_query, query_response):
         try:
             payload = query_response.json()
+            self.raw_query_response = {
```

Comment:
Can you please add a test using normal query response and failure results.

As well as v1/v2 query engine.

Reply:
@xiangfu0 I have added the tests. Please review.